dotCMS/src/main/java/com/dotcms/rest/api/v1/container/ContainerResource.java

```diff
@@ -176,59 +175,63 @@ public final Response containerContent(@Context final HttpServletRequest req, @C
+                            : APILocator.getContentletAPI()
+                                    .find(contentShorty.longId, user, mode.respectAnonPerms);
 
-        if (contentlet.getContentType()
-            .baseType() == BaseContentType.WIDGET) {
-            Map<String, String> response = new HashMap<>();
-            response.put("render", WidgetResource.parseWidget(req, res, contentlet));
+            if (contentlet.getContentType()
+                    .baseType() == BaseContentType.WIDGET) {
+                Map<String, String> response = new HashMap<>();
+                response.put("render", WidgetResource.parseWidget(req, res, contentlet));
 
-            return Response.ok(response)
-                .build();
+                return Response.ok(response)
+                        .build();
 
 
-        }
+            }
 
-        ShortyId containerShorty = APILocator.getShortyAPI()
-            .getShorty(containerId)
-            .orElseGet(() -> {
-                throw new ResourceNotFoundException("Can't find Container:" + containerId);
-            });
+            ShortyId containerShorty = APILocator.getShortyAPI()
+                    .getShorty(containerId)
+                    .orElseGet(() -> {
+                        throw new ResourceNotFoundException("Can't find Container:" + containerId);
+                    });
 
-        Container container = (containerShorty.type != ShortType.IDENTIFIER)
-                ? APILocator.getContainerAPI()
+            Container container = (containerShorty.type != ShortType.IDENTIFIER)
+                    ? APILocator.getContainerAPI()
                     .find(containerId, user, mode.showLive)
-                : (mode.showLive) ? (Container) APILocator.getVersionableAPI()
-                    .findLiveVersion(containerShorty.longId, user, mode.respectAnonPerms)
-                        : (Container) APILocator.getVersionableAPI()
-                            .findWorkingVersion(containerShorty.longId, user, mode.respectAnonPerms);
-
+                    : (mode.showLive) ? (Container) APILocator.getVersionableAPI()
+                            .findLiveVersion(containerShorty.longId, user, mode.respectAnonPerms)
+                            : (Container) APILocator.getVersionableAPI()
+                                    .findWorkingVersion(containerShorty.longId, user,
+                                            mode.respectAnonPerms);
 
+            org.apache.velocity.context.Context context = VelocityUtil.getWebContext(req, res);
 
-        org.apache.velocity.context.Context context = VelocityUtil.getWebContext(req, res);
+            context.put(ContainerLoader.SHOW_PRE_POST_LOOP, false);
+            context.put("contentletList" + container.getIdentifier() + uuid,
+                    Lists.newArrayList(contentlet.getIdentifier()));
+            StringWriter out = new StringWriter();
 
+            VelocityUtil.getEngine()
+                    .mergeTemplate(
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "mergeTemplate"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)